source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/display.tsx

```diff
@@ -48,23 +56,27 @@ class Header extends React.Component<Props & RouteComponentProps<{}>, State> {
+                  <span className='black-text font-lg'>
```

Comment:
Changed this because Josh said that we'll just use the first and last initials of a person's name if they don't have a profile picture.